tests/unit_tests/EvalTests.cpp

```diff
@@ -15,4 +15,12 @@ TEST_CASE("Simple eval", "[eval") {
     CHECK(value.integer() == 13);
 }
 
+TEST_CASE("Function call", "[eval") {
```

Comment:
well, there is at least one list of statements you can test that I am not sure if is being handled properly now...

~~~
return 2;
return 1;
~~~